currency_rate_update_cmc/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```

Comment:
Readme to be split but you know that better than me :)

Reply:
I know. But my only objective at this moment if making this module works.

Reply:
👍